types/filters/parse.go

```diff
@@ -8,6 +8,8 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+
+	versionpkg "github.com/docker/docker/pkg/version"
```

Comment:
Thanks, that was my intention too, fixing it now. I mentioned it in an earlier comment, maybe it got suppressed.